Mathlib/MeasureTheory/Measure/Haar/InnerProductSpace.lean

```diff
@@ -100,3 +103,49 @@ theorem PiLp.volume_preserving_equiv_symm : MeasurePreserving (WithLp.equiv 2 (
   (EuclideanSpace.volume_preserving_measurableEquiv ι).symm
 
 end PiLp
+
+namespace EuclideanSpace
+
+open BigOperators ENNReal
+
+-- See: https://github.com/leanprover/lean4/issues/2220
+local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y)
```

Comment:
Same here: please copy the standard hack + comment, and put it at the top of the file.